onnxruntime/core/providers/webnn/builders/impl/clip_op_builder.cc

```diff
@@ -70,6 +70,9 @@ bool ClipOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializers,
                                       const Node& node,
                                       const WebnnDeviceType /* device_type */,
                                       const logging::Logger& logger) const {
+  // TODO: Update IsOpSupportedImpl to pass GraphViewer instead of InitializedTensorSet so the implementations
+  // can ensure initializers are constant. See #19401 for details of how this update was made to the NNAPI EP.
+  // GetClipMinMax(graph_viewer, node, minValue, maxValue, logger)
   float min, max;
   return GetClipMinMax(initializers, node, min, max, logger);
```

Comment:
is this the only usage of the deprecated one?

Reply:
I believe so. Or the only usage that fails a CI. 

I think the 3 EPs based on the 'builder' approach are NNAPI, CoreML, and WebNN.